### PR TITLE
Fix for issue #9

### PR DESCRIPTION
**Pull request by osbjmg**
I still have one test output in here, I guess I should get a test server to try this stuff out.  However, here we go with the review process anyway.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -9,14 +9,14 @@
 $change='';
 $percentChange='';
 $theTime='';
+$theDate='';
```
Comment (dathelen) [inline index.php]: Hmm yet another variable in the long list of variables.  We need to  look into this at a future date, but I will allow this for now

File: index.php [hunk 2]
```diff
@@ -66,6 +69,9 @@
  echo '<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />';
  echo '<meta http-equiv="refresh" content="305" />';
  echo '<meta property="og:site_name" content="'.$this_site.'" />';
+ echo '<meta name="twitter:label1" value="Reading time"><meta name="twitter:data1" value="1 min read">';
+ #echo '<meta name="twitter:label2" value="Updated"><meta name="twitter:data2" value="'.$theDate.'">';
+ echo '<meta property="article:published_time" content="2017-02-16T16:03:52+00:00" />';
```
Comment (dathelen) [inline index.php]: typically we shouldn't be hardcoding dates here.  I understand that testing this is difficult due to Slack caching things.  But we can't be blaming tools for bad coding practices.

However, in the effort of not stopping progress I will allow it this once

File: index.php [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
+$theDate='';
 $pos_or_neg='';
 $this_site='';
 $this_site_number='';
 $above_or_below='';
 $good='';
 $bad='';
-?>
-<?php
+
```
Comment (dathelen) [inline index.php]: thanks for taking care of this mess

File: index.php (hunk 1 above)
Comment (osbjmg) [inline index.php]: I will bring that up with the PHP Hypertext Processor committee on why they are so hell-bent on "variable declaration".  Python would be more fun here, but yea maybe let's push that to a lunch discussion.

File: index.php (hunk 2 above)
Comment (osbjmg) [inline index.php]: Thanks, and I will work on a dev sandbox site for changes like this from this point forward.

File: index.php (hunk 3 above)
Comment (osbjmg) [inline index.php]: "this mess" wow.

**Pull request merged**